hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -247,6 +247,10 @@ def update_vehicle_with_cached_state(self, token: Token, vehicle: Vehicle) -> No
             else:
                 self._update_vehicle_drive_info(vehicle, state)
 
+    def check_last_action_status(self, token: Token, vehicle: Vehicle, action_id: str) -> bool:
```

Comment:
I would remove this and update this so any API that doesn't have this feature will still work: 

https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/blob/464656e213cc7470b0609f4ed9850ab1229a96c2/hyundai_kia_connect_api/ApiImpl.py#L60-L62

Reply:
In EU I made check_last_action_status always return True, this should make Coordinator.async_wait_for_action_completed work even if the feature isn't implemented in the API. Maybe you're thinking of a better solution?

Reply:
More referencing we also have hyundai usa which doesn't have a stub out so by doing it in the parent location it will cover the usa.

Reply:
Thanks for taking care of that!